torch/quantization/fx/quantization_patterns.py

```diff
@@ -454,8 +457,11 @@ def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_
         emb = quantizer.modules[emb_node.target]
         qconfig = quantizer.qconfig_map[node.name]
         dtypes = get_qconfig_dtypes(qconfig)
-        assert dtypes in supported_dtypes, "qconfig dtype pair not supported:" \
-            " {}, supported dtypes are: {}".format(dtypes, supported_dtypes)
+        if dtypes not in supported_dtypes:
```

Comment:
Could you also update the comments on line 448-449 to update compute_type to float32?